### ISSUE TITLE
False positive for dynamic layers

### ISSUE THREAD

**Issue by priv-kweihmann** [priv-kweihmann/oelint-adv#382]
the following structure

```
- dynamic-layers
  - layer-a
    - a_%.bbappend
  - layer-b
    - a_%.bbappend
```

with both `a_%.bbappend` containing

```
DEPENDS += "foo"
```

gives me the false positive of `oelint.vars.duplicate: Item 'foo' was added multiple times to DEPENDS`, which is a false positive.
Dynamic layers should be handled each a individual groups to be checked
